weasyl/journal.py

```diff
@@ -20,7 +20,7 @@
 
 from libweasyl import text, staff
 
-from weasyl import media
+from weasyl import media, orm
```

Comment:
It makes reordering them easier and insertions or deletions a one-line diff, and stops lines from getting too long (if you add so many to one line that they require wrapping, things start looking less clean).